docs/libcurl/curl_version_info.3

```diff
@@ -216,6 +221,14 @@ backends). (Added in 7.21.4)
 libcurl was built with support for Unix domain sockets.
 (Added in 7.40.0)
 .RE
+
+The use of this field is deprecated as not all features may be represented in
+the bitmask. Use \fIfeature_names\fP instead.
+
+\fIfeature_names\fP is a pointer to an array of string pointers, containing the
+names of the features that libcurl supports. The array is terminated by a NULL
+entry.
```

Comment:
This mentioned that `features` is deprecated, but the man page still only documents the bits in that bitmask. It says nothing about the potential names in `feature_names` that users of this function probably are very interested in. The list above should probably list both?

Reply:
Yes, I can do it. Note this means the man page will still have to be updated when a new feature is introduced.

Reply:
That was always the case. How is a user supposed to know what the feature means if it isn't documented?

Reply:
As `features` is deprecated, do you expect me to remove the bit definitions in the doc ?

Reply:
I think it is way too early for that. The documentation also needs to work as a resource for people reading existing code, and there will be code using this flags for the foreseeable future and then the documentation needs to describe them.

Reply:
> I think it is way too early for that.

Then the doc should be ready now !